site/content/docs/main/config/gateway-api.md

```diff
@@ -0,0 +1,214 @@
+# Gateway API
+
+## Introduction
+
+[Gateway API][1] is an open source project managed by the SIG Network community.
+It is a collection of resources that model service networking in Kubernetes.
+These resources - GatewayClass, Gateway, HTTPRoute, TCPRoute, Service, etc - aim to evolve Kubernetes service networking through expressive, extensible, and role-oriented interfaces that are implemented by many vendors and have broad industry support.
+
+Contour implements Gateway API in addition to supporting HTTPProxy and Ingress.
+In particular, Contour aims to support all [core and extended features][2] in Gateway API.
+
+Gateway API has a comprehensive [website and docs][1], so this document focuses primarily on unique aspects of Contour's Gateway API implementation, rather than attempting to reproduce all of the content available on the Gateway API website.
+The reader is suggested to familiarize themselves with the basics of Gateway API before continuing with this doc.
+
+In Contour's Gateway API implementation, a Gateway corresponds 1:1 with a single deployment of Contour + Envoy.
+In other words, each Gateway has its own control plane (Contour) and data plane (Envoy).
+
+See the [Gateway API guide][12] for a step-by-step tutorial on getting Contour installed with Gateway API and using it to route traffic to a service.
```

Comment:
suggestion to separate the concerns of these two documents for the intended reader

```suggestion
The remainder of this document delves into more detail regarding configuration options when using Contour with Gateway API.
If you are looking for a way to get started with Gateway API and Contour, see the [Gateway API guide][12], a step-by-step tutorial on getting Contour installed with Gateway API and using it to route traffic to a service.
```